src/Components/CuriosityHero/CuriosityHero.scss

```diff
@@ -0,0 +1 @@
+//Please read README section on Styling before editing or creating SCSS
```

Comment:
I probably wouldn't bother putting this on all of our CSS files.
Its going to be a bit annoying adding and reading it at the top of every file! 

I'll just enforce it through code review! :)